server/README.adoc

```diff
@@ -28,6 +28,28 @@ KROKI_DOT_BIN_PATH:: Path to the `dot` binary (provided by GraphViz)
 KROKI_NOMNOML_BIN_PATH:: Path to the `nomnoml` binary
 KROKI_BLOCKDIAG_HOST:: The host of the blockdiag server
 KROKI_BLOCKDIAG_PORT:: The port of the blockdiag server
+KROKI_SAFE_MODE:: One of:
++
+--
+`SECURE`:: (default) Paranoid sanitization of requests before passing to image server.
+`SAFE`:: Assume the image servers secure mode request sanitization is sufficient
+`UNSAFE`:: Run the image servers without request sanitization.
+--
++
+[NOTE]
+--
+Some of the image servers allow referencing external entities by URL or accessing resources from the filesystem.
+
+For example PlantUML allows the `!import` directive to pull fragments from the filesystem or a remote URL or the standard library.
+
+It is the responsibility of the upstream codebases to ensure that they can be safely used without risk.
+Because Kroki does not perform code review of these servies, our default setting is to be paranoid and block imports unless known safe.
```

Comment:
```suggestion
Because Kroki does not perform code review of these services, our default setting is to be paranoid and block imports unless known safe.
```